templates/taxi/driver_detail.html

```diff
@@ -4,7 +4,12 @@
   <h1>
     Username: {{ driver.username }}
   </h1>
-
+  <a href="{% url 'taxi:driver-delete' pk=driver.id %}" class="btn btn-danger link-to-page">
+      Delete
+    </a>
```

Comment:
Code Style: According to the checklist, apart from the 'Delete' button, there should be a 'Cancel' button. This button should lead the user back to the previous page they were on.